### ISSUE TITLE
Sinon 2 or sinon-test support?

### ISSUE THREAD

**Issue by alexmiddeleer** [csantero/ember-sinon#82]
Hey, I was wondering if it was possible to use sinonjs/sinon v2 or sinonjs/sinon-test with this package? My apologies if this is a dumb question 😄 . I believe I am running into [this issue](https://github.com/sinonjs/sinon/issues/1119) (sinon 1.*
 does not work well with qunit promises)

**Comment by elwayman02** [comment]
@alexmiddeleer `master` already has sinon v2, you can try it out today! Hoping to have a new release soon, just need to upgrade a few other packages.

**Comment by elwayman02** [comment]
@alexmiddeleer if you are using ember-sinon-qunit on top of this addon, I believe this issue is resolved by the wrappers in that package: https://github.com/elwayman02/ember-sinon-qunit/issues/55

**Comment by alexmiddeleer** [comment]
TLDR: Thanks for the quick response (just as I was typing this, in fact), though my issue was unrelated.

I didn't realize that the component isn't reinitialized for every test, so mocking an imported dependency (say, `import bar from 'somewhere'`) won't work  if you do `const foo = dependency.bar` in your component. Since test'em reruns the broken test first, the tests pass every other time 😅
